Gemfile

```diff
@@ -2,6 +2,8 @@ source 'http://rubygems.org'
 
 gemspec
 
+gem 'launchy'
```

Comment:
This should only go into `.gemspec`, so that it gets installed with `bundle install`.